contextily/place.py

```diff
@@ -178,13 +185,7 @@ def __repr__(self):
 
 
 def plot_map(
-    place,
-    bbox=None,
-    title=None,
-    ax=None,
-    axis_off=True,
-    latlon=True,
-    attribution=ATTRIBUTION,
+    place, bbox=None, title=None, ax=None, axis_off=True, latlon=True, attribution=None
```

Comment:
This is not `black` compatible?

Reply:
It's black that did it, so I suppose yes :)

(I think the rule is: if it fits on one line, use one line, if not, put each keyword on its own line)